content/lessons/chapter-7/outro-1.tsx

```diff
@@ -4,8 +4,8 @@ import { useTranslations } from 'hooks'
 import { ChapterEnd } from 'ui'
 
 export const metadata = {
-  title: 'chapter_four.outro_one.title',
-  image: '/assets/images/chapter-4-finish.jpg',
+  title: 'chapter_seven.outro_one.title',
+  image: '/assets/images/chapter-7-outro.jpg',
   theme: 'bg-transparent/20',
   secondaryTheme: 'bg-[#472B26]',
   gradientTheme: 'from-[#472B26] via-[rgba(71, 43, 38, 0)]',
```

Comment:
Lets make the color used be this `#52555d` instead

Reply:
so for the second line that would be `from-[#52555d] via-[#52555dBF]`

Reply:
the secondary theme?